src/test/java/bdd/steps/StepDefinitions.java

```diff
@@ -0,0 +1,33 @@
+package bdd.steps;
+
+import driver.DriverManager;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import pages.cucumber.CucumberToolsPage;
+import pages.cucumber.CucumberHomePage;
+
+
+public class StepDefinitions {
+
+    @When("the user opens Cucumber website")
+    public void openCucumberWebsite() {
+        new CucumberHomePage(DriverManager.getDriver()).openPage();
+    }
+
+    @When("the user clicks on the {string} section")
+    public void clickOnSection() {
+        new CucumberHomePage(DriverManager.getDriver()).menuDropdownSection.click();
+    }
+
+    @When("the user clicks on the {string} item")
+    public void clickOnItem() {
```

Comment:
...(String item)...

Reply:
Refactored, thank you.
Please see in the following commit.